.drone.star

```diff
@@ -2061,10 +2060,6 @@ def ocisServer(storage, accounts_hash_difficulty = 4, volumes = [], depends_on =
         environment["APP_PROVIDER_WOPI_WOPI_SERVER_EXTERNAL_URL"] = "http://wopiserver:9300"
         environment["APP_PROVIDER_WOPI_FOLDER_URL_BASE_URL"] = OCIS_URL
 
-    if deploy_type == "federation":
-        environment["OCIS_URL"] = OCIS_FED_URL
-        container_name = "federation-ocis-server"
```

Comment:
We should use this and remove the one below. This will work well with ociswrapper command and step names
```py
 if deploy_type == "federation":
        environment["OCIS_URL"] = OCIS_FED_URL
        environment["PROXY_HTTP_ADDR"] = OCIS_FED_DOMAIN
        container_name = "federation-ocis-server"
```

Reply:
I put it after `ocis` to redefine `ocis["name"]`

Reply:
` container_name = "federation-ocis-server"` should do the job